src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -37,20 +34,20 @@
  * The relevant fields for the database, which are described in that class are:
  *    - Kind: "revision"
  *    - Key: commit hash
- * The database indexes after all the BuildInfo fields, besides the {@code builders} list.
- * It is unfeasable to query after the list of builders, since it would not be possible to
- * replicate such a list without querying the database for it first, and the complexity
- * of the indexing and querying would be great, without any apparent value.
  *
  * Useful links:
  * - https://googleapis.dev/java/google-cloud-datastore/latest/index.html
  * - https://googleapis.dev/java/spring-cloud-gcp/1.2.2.RELEASE/index.html
  * - https://cloud.google.com/datastore/docs/concepts/
  * - https://spring.io/projects/spring-cloud-gcp#overview
  */
-@SpringBootApplication
-public class GCDataRepository implements DataRepository {
+@Repository
+public class DatastoreRepository implements DataRepository {
+  @Autowired
   private DatastoreTemplate storage;
+  private final ScheduledExecutorService cleanScheduler = Executors.newScheduledThreadPool(1);
+  private final DataRetentionParameters dataRetentionParameters;
```

Comment:
I believe we don't need the `cleanScheduler` and the `dataRetentionParameters` members anymore.